docs/source/content/overview/linear-regression.rst

```diff
@@ -6,81 +6,83 @@ Linear Regression
   :local:
   :depth: 3
 
+*******
+Summary
+*******
+Linear regression is a technique used to analyze a linear relationship between input variables and a single output variable. We measure how accurate the resulting model is using a cost function. We want to minimize the value of the cost function in order to get a more accurate model.
+
 ********
 Overview
 ********
-**Linear regression** is a technique used to analyze a **linear relationship** between **input/independent/x** variables and a single **output/dependent/y** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
-
-.. image:: _img/LR.png
+**Linear regression** is a technique used to analyze a **linear relationship** between **input** variables and a single **output** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
 
-Our goal is to find the line that best models the path of the data points called a line of best fit [*Equation 1*].
+.. figure:: _img/LR.png
+   
+   *Figure 1*. A sample data set with a linear relationship
```

Comment:
I'm not sure how these figure captions will turn out on the site, but in github they're a bit hard to distinguish from other text. Can we make them a smaller font or bold or some other way to distinguish them?